src/pal/prebuilt/inc/clrallevents.h

```diff
@@ -0,0 +1,2218 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// Do not modify the file. This file is generated using the script from <root>/src/inc/genXplatEtw.pl
+//
+
+extern "C" BOOL  EventEnabledGCStart();
+extern "C" ULONG   FireEtwGCStart(const unsigned int  Count,
```

Comment:
I will add the 4 spaces,.
What is wrong in the 2 spaces between type and the name of the argument. That is kind of intentional and keeps the script simple, as it shares code between pointer types and non pointer types